site/_data/refdocs/config.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
should this content be added? the hugo site format doesnt have a `_data` dir so this seems entirely new

Reply:
It's used for the generator to build the site. The _data dir doesn't get published over to the hugo site from my tests.

Reply:
from the generate-api-docs.sh script looks like these templates shouldnt have been deleted, maybe we can modify that script to look for them somewhere else to keep with the hugo structure?